aiida_common_workflows/cli/options.py

```diff
@@ -237,3 +237,7 @@ def convert(self, value, param, ctx):
     required=False,
     help='An instance of a completed workchain of the same type as would be run for the given plugin.'
 )
+
+OUTPUT_FILE = options.OverridableOption(
+    '-o', '--output-file', type=click.STRING, required=False, help='Save the plot to a specified file.'
```

Comment:
```suggestion
    '-o', '--output-file', type=click.STRING, required=False, help='Save the output to a specified file.'
```
Maybe to make the help string more widely usable? i.e. when you write data in table-form instead of plotting it

Reply:
Very true! Made the option before adapting it so it could also be used for tables, as per your (excellent) suggestion.